### PR TITLE
Clean up code

### PR DESCRIPTION
**Pull request by melisabok**
- Remove unused process_dir and webfield_dir parameters
- Don't set a cdate, let's leave that to ther server, remove epoche_time function
- Remove result sorting for groups and invitations. The response is already sorted by date, if you need another sorting we should implement it in the backend.
- Make mandatories fields for Group, Invitation, Tag and Notes
- Remove forum, reply, invitation from the Invitation constructor, they should be in the reply dictionary
- Add tcdate and tmdate that were missing in the classes.

### CODE REVIEW

**Comment by melisabok** [comment]
Yes, I think we should fix the scripts. We need to have a solid library. Let me update the version number

File: openreview/openreview.py [hunk 1]
```diff
@@ -8,16 +8,19 @@
...
-
 class OpenReviewException(Exception):
     pass
 
 class Client(object):
 
-    def __init__(self, baseurl = None, process_dir = '../process/', webfield_dir = '../webfield/', username = None, password = None):
-        """CONSTRUCTOR DOCSTRING"""
+    def __init__(self, baseurl = None, username = None, password = None):
+        """
```
Comment (mohituniyal) [inline openreview/openreview.py]: Just a note: I do not think this doc-string will show up in the documentation.

Comment (melisabok) [reply openreview/openreview.py]: Do you know why?

I copied the example from Elastic Search:

https://github.com/elastic/elasticsearch-py/blob/master/elasticsearch/client/__init__.py#L174

Comment (mohituniyal) [reply openreview/openreview.py]: Sorry, looks like it does show up. I made an incorrect assumption that any function with a leading underscore does not show up in the documentation.

**Pull request merged**